types/react/v15/index.d.ts

```diff
@@ -223,7 +223,7 @@ declare namespace React {
         props?: ClassAttributes<T> & P,
         ...children: ReactNode[]): CElement<P, T>;
     function createElement<P>(
-        type: ComponentClass<P>,
+        type: SFC<P> | ComponentClass<P>,
```

Comment:
seems reasonable to have this catch-all case also take `string`. Do you mind adding that?

Reply:
Just added string.